cms/djangoapps/contentstore/views/component.py

```diff
@@ -294,14 +307,19 @@ def create_support_legend_dict():
             "documentation_label": _("{platform_name} Support Levels:").format(platform_name=settings.PLATFORM_NAME)
         }
 
+    # @medality_custom
     component_display_names = {
-        'discussion': _("Discussion"),
+        'ambra': _("Ambra Link"),
+        'ambra_case_list': _("Ambra Case List"),
+        'ambra_quiz': _("Ambra Quiz"),
+        'drag_and_drop': _("Drag and Drop"),
         'html': _("Text"),
+        'pdf': _("PDF"),
         'problem': _("Problem"),
-        'video': _("Video"),
-        'openassessment': _("Open Response"),
-        'library': _("Library Content"),
-        'drag-and-drop-v2': _("Drag and Drop"),
+        'select_from_library': _("Select from Library"),
+        'typeform': _("Typeform Embed"),
```

Comment:
Nice! We can close https://mrionline.atlassian.net/browse/MRI-5978 as not doing with this